confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/input/ConfluenceInputFilterStream.java

```diff
@@ -417,7 +457,8 @@ private void readPage(long pageId, String spaceKey, Object filter, ConfluenceFil
 
         // Skip pages with empty title
         if (StringUtils.isEmpty(documentName)) {
-            this.logger.warn("Found a page without a name or title (id={}). Skipping it.", pageId);
+            this.logger.error("Found a page without a name or title (id={}). Skipping it.",
```

Comment:
Error is a serious problem that should really not be ignored, and I'm not sure this use case really qualify.